lib/IRGen/GenType.cpp

```diff
@@ -1888,13 +1888,7 @@ const TypeInfo *TypeConverter::convertType(CanType ty) {
   PrettyStackTraceType stackTrace(IGM.Context, "converting", ty);
 
   switch (ty->getKind()) {
-  case TypeKind::Error: {
```

Comment:
Removing this should actually fail an existing test...

Reply:
Ya, that test is [needlessly?] macOS only, so I didn't trip over the failure.

Reply:
Do you mind removing the MacOS dependency from the test?